### ISSUE TITLE
这儿感觉代码风格不是很好

### ISSUE THREAD

**Issue by azhsmesos** [sofastack/sofa-ark#516]
`@Override
    @SuppressWarnings("unchecked")
    public boolean registerBiz(Biz biz) {
        AssertUtils.assertNotNull(biz, "Biz must not be null.");
        AssertUtils.isTrue(biz.getBizState() == BizState.RESOLVED, "BizState must be RESOLVED.");
        bizRegistration.putIfAbsent(biz.getBizName(), new ConcurrentHashMap<String, Biz>(16));
        ConcurrentHashMap bizCache = bizRegistration.get(biz.getBizName());   // test
        return bizCache.put(biz.getBizVersion(), biz) == null;
    }`
比如这儿，SuppressWarnings完全可以不要，只需要把ConcurrentHashMap返回值改成 ConcurrentHashMap<String, Biz>就好了

该方法在BizManagerServiceImpl类中，我感觉应该要尽量少出现@SuppressWarnings注解比较好

**Comment by azhsmesos** [comment]
![image](https://user-images.githubusercontent.com/65518067/171790518-c74aaf34-e706-4b8c-aba4-0559942c212b.png)

**Comment by glmapper** [comment]
@azhsmesos that's right, there may be other similar problems, If you are interested in this issue, please submit your PR

**Comment by azhsmesos** [comment]
已经提pr了麻烦review下~
